packages/components/src/components/PipelineRuns/PipelineRuns.stories.jsx

```diff
@@ -14,8 +14,9 @@ limitations under the License.
 
 import { getStatus } from '@tektoncd/dashboard-utils';
 import { action } from '@storybook/addon-actions';
-import { TrashCan32 as Delete } from '@carbon/icons-react';
-import { Dropdown } from 'carbon-components-react';
+import { TrashCan as Delete } from '@carbon/react/icons';
```

Comment:
Not persisting the size here?

Reply:
batch action icons are constrained to 16px anyway, not sure why we used a bigger size before but 16px is expected now